src/components/TooltipGlobal/TooltipGlobal.test.js

```diff
@@ -0,0 +1,264 @@
+import { queryHelpers, render, screen, fireEvent } from '@testing-library/react';
+
+import { MockJetsContextProvider } from '../../__mocks__/MockJetsContext';
+
+import { JetsTooltipGlobal } from './index';
+
+import { activeTooltipData, paramsData } from './__fixtures__';
+import { userEvent } from '@storybook/testing-library';
+
+const setup = ({ data = {}, config = {}, params = {}, events = {} } = {}) => {
+  render(
+    <MockJetsContextProvider config={config} params={{ ...paramsData(params) }} events={events}>
+      <JetsTooltipGlobal data={{ ...activeTooltipData(data) }} />
+    </MockJetsContextProvider>
+  );
+};
+
+describe('JetsTooltipGlobal', () => {
+  // General content and functionality
+  it('applies classes for horizontal layout', async () => {
```

Comment:
It would be good to maintain consistency in how we describe the tests. Could we follow the "when/should" pattern that Lindy applied in the unit tests, please? I've seen the same approach used in some of our microfrontends. e.g: should apply classes for horizontal layout

Reply:
I've refactored to match the when/should pattern 👍🏽